OpenRA.Game/Platform.cs

```diff
@@ -94,12 +94,26 @@ static string GetSupportDir()
 			return dir + Path.DirectorySeparatorChar;
 		}
 
-		public static string GameDir { get { return AppDomain.CurrentDomain.BaseDirectory; } }
+		public static string GameDir
+		{
+			get
+			{
+				var gameDir = AppDomain.CurrentDomain.BaseDirectory;
+
+				// BUG: In Game.cs we get GameDir with trailing DirectorySeparator, but not in PlatformTest.cs
+				if (!gameDir.EndsWith(Path.DirectorySeparatorChar.ToString()))
```

Comment:
I suggest `if (gameDir[gameDir.Length - 1] != Path.DirectorySeparatorChar)`.  If you insist on using `EndsWith` (but IMO having to ToString the separator char is ugly) then you must include the `StringComparison.Ordinal`.